### PR TITLE
add README

### PR DESCRIPTION
**Pull request by chukarave**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Connecting Senses
+
+A tool for connecting Lexeme Senses with the corresponding Wikidata items.
```
Comment (micgro42) [inline README.md]:
```suggestion
A tool for connecting Lexeme Senses with the corresponding Wikidata Items.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Usage
+
+Authentication is done via OAuth with Wikidata.
+
+Select an Item for the language in which you would like to receive senses.
+You will then receive a Lemma and a Sense. Search for the corresponding wikidata item in the search field.
```
Comment (micgro42) [inline README.md]:
```suggestion
You will then receive a Lemma and a Sense. Search for the corresponding Wikidata Item in the search field.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Note: This tool uses Node 10 and npm 6.14.
+
+- create a local `.env` file (see `.env.example` for reference)
+- In order to start your local development environment, first run `npm i` to install dependencies.
+- For the frontend environment run:
+    `npm run serve`
+- For backend development run:
+    `npx nodemon --inspect server.js`
+Per default, the tool will be running locally at `http://localhost:8080/ `.
```
Comment (micgro42) [inline README.md]: This is true for the frontend dev server, the backend dev server runs at `port 5000`, I think, or whatever is in the PORT env variable.

**Review by micgro42 — COMMENTED** [review]
Thank you! Some minor suggestions

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
```
Comment (itamargiv) [inline README.md]:
```suggestion
Per default, the frontend development server will be running locally at `http://localhost:8080/ `, and the backend development server will run at port `5000`.
```

**Pull request merged**